### PR TITLE
Release 15/06/2017

### PR DESCRIPTION
**Pull request by attiks**
Update features

### CODE REVIEW

File: html/sites/all/modules/custom/events_event/events_event.strongarm.inc [hunk 1]
```diff
@@ -110,13 +302,25 @@ function events_event_strongarm() {
       'display' => array(
         'ev_local_time' => array(
           'default' => array(
-            'weight' => '7',
-            'visible' => TRUE,
+            'weight' => '19',
+            'visible' => FALSE,
```
Comment (emmajane) [inline html/sites/all/modules/custom/events_event/events_event.strongarm.inc]: Should this have been flipped to FALSE?

File: html/sites/all/modules/custom/events_event/events_event.features.inc [hunk 2]
```diff
@@ -29,7 +32,7 @@ function events_event_views_api($module = NULL, $api = NULL) {
 function events_event_node_info() {
   $items = array(
     'ev_event' => array(
-      'name' => t('ev_event'),
+      'name' => t('Event'),
```
Comment (emmajane) [inline html/sites/all/modules/custom/events_event/events_event.features.inc]: Yay!

File: html/sites/all/modules/custom/events_event/events_event.features.field_instance.inc [hunk 3]
```diff
@@ -775,6 +776,7 @@ function events_event_field_default_field_instances() {
 
   // Translatables
   // Included for use with string extractors like potx.
+  t('Additional Information');
```
Comment (emmajane) [inline html/sites/all/modules/custom/events_event/events_event.features.field_instance.inc]: Yay!

File: html/sites/all/modules/custom/events_event/events_event.features.field_instance.inc [hunk 4]
```diff
@@ -83,7 +83,7 @@ function events_event_field_default_field_instances() {
         'label' => 'above',
         'module' => 'taxonomy',
         'settings' => array(),
-        'type' => 'taxonomy_term_reference_link',
+        'type' => 'taxonomy_term_reference_plain',
```
Comment (emmajane) [inline html/sites/all/modules/custom/events_event/events_event.features.field_instance.inc]: this is now "plain" not "link" .. assume it's right this changed?

File: html/sites/all/modules/custom/events_event/events_event.info [hunk 5]
```diff
@@ -11,6 +12,8 @@ dependencies[] = date_popup
 dependencies[] = date_repeat
 dependencies[] = date_repeat_field
 dependencies[] = date_views
+dependencies[] = ds
+dependencies[] = ds_extras
```
Comment (emmajane) [inline html/sites/all/modules/custom/events_event/events_event.info]: yay! but does the module also get installed somewhere? I'm rusty on dependencies. Should there be a hook update for that in a custom module to make sure it's turned on?

File: html/sites/all/modules/custom/events_event/events_event.features.field_instance.inc (hunk 4 above)
Comment (attiks) [inline html/sites/all/modules/custom/events_event/events_event.features.field_instance.inc]: Yes

File: html/sites/all/modules/custom/events_event/events_event.info (hunk 5 above)
Comment (attiks) [inline html/sites/all/modules/custom/events_event/events_event.info]: It's automagically

File: html/sites/all/modules/custom/events_event/events_event.strongarm.inc (hunk 1 above)
Comment (attiks) [inline html/sites/all/modules/custom/events_event/events_event.strongarm.inc]: It's like that on staging, there's another output field for the times

**Pull request merged**